go/vt/mysqlctl/binlogs_gtid.go

```diff
@@ -108,6 +107,15 @@ func ChooseBinlogsForIncrementalBackup(
 		if err != nil {
 			return nil, "", "", vterrors.Wrapf(err, "cannot evaluate incremental backup from pos")
 		}
+		if incrementalBackupFromGTID == "" {
+			// This can happen on the very first binary log file. It happens in two scenarios:
+			// 1. This is the first binlog ever in the history of the mysql server; the GTID is truly empty
+			// 2. A full backup was taken and restored, with all binlog scraped.
```

Comment:
Did you mean `binlog scrapped` here, meaning deleted? Or maybe you mean scraped as in read/applied?